src/js/apps/patients/sidebar/filters-sidebar_app.js

```diff
@@ -0,0 +1,45 @@
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+import { LayoutView, GroupsFilterView } from 'js/views/patients/sidebar/filters/filters-sidebar_views';
+
+export default App.extend({
+  stateEvents: {
+    'change': 'onChangeFilters',
+  },
+  onStart({ state }) {
+    this.currentClinician = Radio.request('bootstrap', 'currentUser');
+
+    this.showView(new LayoutView());
+
+    this.showGroupsFilterView(state);
+  },
+  viewEvents: {
+    'close': 'stop',
+    'click:clear:filters': 'onClearFilters',
+    'select:new:group': 'onSelectNewGroup',
+  },
+  onChangeFilters(state) {
+    this.showGroupsFilterView(state.attributes);
+  },
+  showGroupsFilterView(state) {
+    const groups = this.currentClinician.getGroups();
+
+    const groupsFilterView = new GroupsFilterView({
+      groups,
+      state,
```

Comment:
I think maybe this could change to `groupId: this.getState('groupId'),` instead of passing the state around

Reply:
Also I think rather than pass down the event because the app is instantiating it here this could be
```js
this.listenTo(groupsFilterView, 'select:group', this.onSelectGroup)` or just inlining the handler in this case'd be fine too.

```

Reply:
Nice, I like it. Updated in the last push.